### PR TITLE
Test against `Ruby 3.1` and `Ruby 3.3`

### PR DESCRIPTION
**Pull request by nicolastemciuc**
## Summary

As tilted, update CI to test against `Ruby 3.1.6` and `Ruby 3.3.6`

### CODE REVIEW

**Comment by nicolastemciuc** [comment]
Maybe in another PR (or this one) we can update the CI to run with the newest OpenSSL that not supports `Ruby < 2.7`

image

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -16,6 +16,8 @@ jobs:
       fail-fast: false
       matrix:
         ruby:
+          - 3.3.6
+          - 3.1.6
```
Comment (santiagorodriguez96) [inline .github/workflows/build.yml:20]: What do you think about adding Ruby 3.2 to the matrix too?

Comment (santiagorodriguez96) [inline .github/workflows/build.yml:20]: I know that we were already doing it this way, but I would also avoid specifying the patch version:

```suggestion
          - 3.3
          - 3.1
```

That way we ensure that we always tun the CI against the latest patch versions. Then, as a followup we can update the rest of the entries of the ruby matrix

**Review by santiagorodriguez96 — COMMENTED** [review]
Cool! Thank you 🙂

**Comment by santiagorodriguez96** [comment]
> Maybe in another PR (or this one) we can update the CI to run with the newest OpenSSL (that not supports Ruby < 2.7)

Makes sense!

File: .github/workflows/build.yml (hunk 1 above)
Comment (nicolastemciuc) [reply .github/workflows/build.yml:20]: Makes sense!

Comment (nicolastemciuc) [reply .github/workflows/build.yml:20]: Good call

Comment (nicolastemciuc) [reply .github/workflows/build.yml]: Done in [1b405fd](https://github.com/cedarcode/cose-ruby/pull/66/commits/1b405fd74c32edf9f4ce4c5f70d73d9934122276)

Comment (nicolastemciuc) [reply .github/workflows/build.yml]: Done in [9c3a465](https://github.com/cedarcode/cose-ruby/pull/66/commits/9c3a4650f6509562f55a3a053d073c606c65358c)

**Review by santiagorodriguez96 — APPROVED** [review]
LGTM! Thank you ❤️

**Pull request merged**